### PR TITLE
feature separate border styles for each side

### PR DESCRIPTION
**Pull request by prpateldev**
- check example column styles
- adding config lineSides in styles
- pass single or array of LineSide
- accepting lineWidth as number and array of number

### CODE REVIEW

**Comment by simonbengtsson** [comment]
Awesome! Instead of introducing lineSides as a new option how about enabling specifying lineWidth as an array or object {left: number, right: number etc} similar to margin and cellPadding. That way it would be possible to specify different widths for different lines as well.

**Comment by prpateldev** [comment]
that's a good idea I will implement it, Hoping to merge soon as Hacktoberfest :D

**Comment by prpateldev** [comment]
Just pushed a commit 
- removed lineSides 
- adding object like {left: number, right: number}

Please check 👍

File: examples/examples.js [hunk 1]
```diff
@@ -276,6 +276,40 @@ examples.colstyles = function () {
...
+      name: {
+        lineColor: '576574',
+        lineWidth: { bottom: 0.5 }
+      },
+      city: {
+        lineColor: '576574',
+        lineWidth: { left: 0.2, bottom: 0.5 }
+      },
+      email: {
+        lineSides: 'bottom',
```
Comment (simonbengtsson) [inline examples/examples.js:300]:
```suggestion
```

File: examples/examples.js [hunk 2]
```diff
@@ -276,6 +276,40 @@ examples.colstyles = function () {
...
+        lineColor: '576574',
+        lineWidth: { bottom: 0.5 }
+      },
+      expenses: {
+        lineSides: 'bottom',
```
Comment (simonbengtsson) [inline examples/examples.js:305]:
```suggestion
```

File: src/tableDrawer.ts [hunk 3]
```diff
@@ -249,11 +249,10 @@ function printRow(doc: DocHandler, table: Table, row: Row, cursor: Pos) {
       continue
     }
 
-    const cellStyles = cell.styles
-    const fillStyle = getFillStyle(cellStyles.lineWidth, cellStyles.fillColor)
-    if (fillStyle) {
-      doc.rect(cell.x, cursor.y, cell.width, cell.height, fillStyle)
-    }
+    // draw cell borders
+    drawBorders(doc, cell, cursor)
```
Comment (simonbengtsson) [inline src/tableDrawer.ts:253]:
```suggestion
    drawCellBorders(doc, cell, cursor)
```

File: src/tableDrawer.ts [hunk 4]
```diff
@@ -277,6 +276,56 @@ function printRow(doc: DocHandler, table: Table, row: Row, cursor: Pos) {
   cursor.y += row.height
 }
 
+function drawBorders(doc: DocHandler, cell: Cell, cursor: Pos) {
```
Comment (simonbengtsson) [inline src/tableDrawer.ts:279]:
```suggestion
function drawCellBorders(doc: DocHandler, cell: Cell, cursor: Pos) {
```

**Review by simonbengtsson — APPROVED** [review]
Awesome! I'm think it's ready for merge. Just had some minor formatting suggestions 👌

**Pull request merged**